PythonTool/PlotPseudocolor.py

```diff
@@ -0,0 +1,51 @@
+from visit import *
+
+
+def PlotPseudocolor(File, ObjectSequence):
+    # Pseudocolor plot attributes.
+
+    # Change orientation of legend
+    LegendAttributes = GetAnnotationObject(ObjectSequence["Pseudocolor"])
+    LegendAttributes.orientation = "VerticalLeft"
+    LegendAttributes.fontHeight = 0.03
+
+    Attribute = PseudocolorAttributes()
+
+    if File is not None:
+        File = list(File)
+
+        for item in File:
+            if item[1].title() == "Pseudocolor":
+
+                # Check for optional inputs in Dictionary input.
+                try:
+                    Attribute.scaling = eval(
+                                             "Attribute." +
+                                             str(item[3]).title()
+                                             )
+                except Exception:
+                    pass
+
+                try:
+
+                    if (item[4])[0].title() == "Min":
```

Comment:
Dictionary inputs would be better. I catch myself in the MATLAB mentality where I choose values whether or not they are in a specific position. I will see what I can do to change this. Calling plots and plot options from dictionaries will most likely be the last thing I attempt to do because it changes many things throughout  written scripts.